eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -108,6 +115,7 @@ steps:
               -publish $(_PublishArgs)
               -configuration ${{ parameters.buildConfig }}
               /bl:${{ parameters.repoLogPath }}/transport.binlog
+              /p:EnableEvaluationReportBuild=true
```

Comment:
I don't believe this is needed here. This step only builds Microsoft.Internal.Extensions.DotNetApiDocs.Transport project.